### DISCUSSION TITLE
Demande de modification de l'année sur l'affiche

### DISCUSSION THREAD

**Discussion by Greta712** [betagouv/ma-cantine#1669] [Retours utilisateur·rice]
Pour le générateur d’affiche, est-il possible de modifier l’année? Pour le moment nous avons que l’année 2021. Pouvons-nous imaginer de permettre à l’utilisateur de pouvoir par exemple saisir la période qu’il souhaite sachant que maintenant ils doivent communiquer de manière permanente, avec les données actualisées je suppose

**Comment by paulineeyherabide** [comment]
@Greta712 j'ai eu un nouveau retour en ce sens-là, afin de pouvoir choisir l'année sur l'affichage :)
